template/ios/fastlane/Fastfile

```diff
@@ -15,18 +15,28 @@
 
 require 'dotenv'
 
+package_json_content = File.read('../../package.json')
+package_json = JSON.parse(package_json_content)
+version = package_json['version']
+
 default_platform(:ios)
 
-platform :ios do
+owner_name = "STS-ReactNative" # Update this value if you are not work on Saigon Technology Solutions
+projectName = ENV["PROJECT_NAME"]
+workspace = projectName + ".xcworkspace"
+xcodeproj = projectName + ".xcodeproj"
+my_app = ENV["APPCENTER_APP_NAME_IOS"]
+teamId = ENV["APPLE_TEAM_ID"]
 
-  owner_name = "Saigontechnology" #Replace owner name here
-  my_app = "ReactNativeBaseTypeScriptIos" #Replace my app here
-  platform = 'ios'
+platform :ios do
+  before_all do |lane, options|
+      env = options[:env].downcase
+      Dotenv.load("../../.env.#{env}")
+  end
 
   desc "Build and upload to TestFlight"
   lane :upload do |options|
     env = options[:env].downcase
```

Comment:
remove this line cuz we have lane before_all

Reply:
removed